http/src/main/resources/init-resources/gce-init.sh

```diff
@@ -105,11 +105,12 @@ export MEM_LIMIT=$(memLimit)
 export WELDER_MEM_LIMIT=$(welderMemLimit)
 export PROXY_SERVER_HOST_NAME=$(proxyServerHostName)
 export WELDER_ENABLED=$(welderEnabled)
-if [ ! -z "$RSTUDIO_DOCKER_IMAGE" ] ; then
-  export IS_RSTUDIO_RUNTIME="true"
-else
-  export IS_RSTUDIO_RUNTIME="false"
-fi
+export IS_RSTUDIO_RUNTIME="false" # TODO: update to commented out code once we release Rmd file syncing
```

Comment:
I'm assuming that since this is a VM-scoped env var that it doesn't have any affect on things such as Leonardo's labelling of the runtime as an RStudio runtime?

Reply:
didn't realize there's labeling in DB for this as well..is that being used anywhere?

Reply:
it doesn't affect the db - just double checked that an rstudio runtime still has the label `tool: RStudio`. not sure where we use this though